src/backend/render/mod.rs

```diff
@@ -664,7 +668,7 @@ where
 
     let offset = match previous.as_ref() {
         Some((previous, previous_idx, start)) => {
-            let layout = state.config.workspace.workspace_layout;
+            let layout = state.config.config.workspaces.workspace_layout;
```

Comment:
I am not really a big fan of the `config.config` throughout the code, I think that is confusing.

Couple of ideas to resolve this:
- Make a separate `TilingConfig` struct instead of querying `CosmicConfConfig` directly.
- Rename the field to `cosmic` (to differentiate from `static` and `dynamic`).

Reply:
Ok, I think I prefer the latter if that is alright with you?

Reply:
Sure. I am just wondering how much this breaks current configs? Not something we need to worry about too much, but this would move all keys into a single cosmic-config file, right?

Reply:
No, each of these keys would be a separate file, but ya, I think it breaks the current configs because it removes the "tiling_enabled" key.

Reply:
The static-config breaking isn't a big issue. Once we move over the key-bindings to something else (likely once dealing with the issue of global keybindings), I want to get rid of it anyway. So this is likely the last time we will break it and just before the alpha is the best time to do that. :laughing: 

I am fine with the second option then.